inc/saiobject.h

```diff
@@ -37,6 +37,9 @@
 #include <saisrv6.h>
 
 /* new experimental object type includes */
+#include <saiexperimentaldashha.h>
```

Comment:
This is not needed here, no changes are made to this file so no object includes are required

Reply:
Thanks Kamil. I have removed these lines now.